tutorials/telegraf-output-plugin.md

```diff
@@ -0,0 +1,314 @@
+# PostgreSQL and TimescaleDB output plugin for Telegraf
+
+Telegraf can collect metrics from a wide array of inputs and write them into a wide array of outputs. It is plugin-driven for both collection and output of data so it is easily extendable. It is written in Go, which means that it is a compiled and standalone binary that can be executed on any system with no need for external dependencies, no npm, pip, gem, or other package management tools required. 
+
+Telegraf is an open-source tool. It contains over 200 plugins for gathering and writing different types of data written by people who work with that data. 
+As an open-source project hosted on GitHub it has a typical Pull Request review process. When a plugin is written it is submitted for a review by the developers of Telegraf. 
+If accepted the new code is merged and the plugin is available with the following release. 
+
+We wrote the PostgreSQL output plugin which also has the ability to send data to an TimescaleDB hypertable. The Pull Request is open and currently under review by the Telegraf developers. 
+
+This tutorial will show a couple of examples on how to use the PostgreSQL/TimescaleDB output plugin for Telegraf. While the Pull Request is in review, we provide a built binary of Telegraf with our plugin already included that users can download.  
+
+## Installation [](telegraf-installation)
+
+### Before we start
+
+Before we start, you will need [TimescaleDB installed][getting-started] and a means to connect to it.
+
+### Setting up Telegraf
+
+Telegraf is written in Go, so only one standalone binary is required to run it. But also because of this all the code for the different plugins must be part of that binary. We have an unofficial build of Telegraf version 1.10.4 with our plugin added. You can download from the following links: 
```

Comment:
First 2 sentences sound a bit weird, also i think go added support for plugins but telegraf does not use that infrastructure yet